tests/ui/lint/non-local-defs/cargo-update.rs

```diff
@@ -0,0 +1,13 @@
+//@ check-pass
+//@ edition:2021
+//@ aux-build:non_local_macro.rs
+//@ rustc-env:CARGO_CRATE_NAME=non_local_def
```

Comment:
Perhaps comment that this is for the suggestions?

Reply:
Sure, done; and also added an `@error-pattern` to make sure we emit the `cargo update` suggestion.